File: ruby/fizz_buzz.rb
n = 15

def fizz_buzz(n)
  answer = []

  (n + 1).times do |i|
    next unless i > 0

    answer << if (i % 3).zero? && (i % 5).zero?
                'FizzBuzz'
              elsif (i % 3).zero?
                'Fizz'
              elsif (i % 5).zero?
                'Buzz'
              else
                i.to_s
              end
  end

  answer
end

p fizz_buzz(n)

File: python/fizz_buzz.py
n = 15


def fizz_buzz(n):
  answer = []

  for i in range(n + 1):
    if i > 0:
      if (i % 3) == 0 and (i % 5) == 0:
        answer.append("FizzBuzz")
      elif (i % 3) == 0:
        answer.append("Fizz")
      elif (i % 5) == 0:
        answer.append("Buzz")
      else:
        answer.append(str(i))


  return answer


print(fizz_buzz(n))